utils/documentOps.py

```diff
@@ -205,7 +205,7 @@ def print_row(array):
 def print_md():
     # Header.
     print("<!--- Automatically generated, do not edit. -->")
-    print("<!--- To update, run `make onnx_mlir_supported_ops` -->")
+    print("<!--- To update, run `make onnx_mlir_supported_ops_"+ target_arch +"' -->")
```

Comment:
Okay I changed this to make it architecture specific and easier to run :)

Reply:
Thanks!